app/assets/javascripts/client/inventories/InventoryDataEntriesCtrl.js

```diff
@@ -0,0 +1,51 @@
+(function() {
+  'use strict';
+  angular.module('PDRClient')
+      .controller('InventoryDataEntriesCtrl', InventoryDataEntriesCtrl);
+
+  InventoryDataEntriesCtrl.$inject = [
+    '$scope',
+    '$q',
+    'DataEntry',
+    'DTOptionsBuilder',
+    'DTColumnBuilder'
+  ];
+
+  function InventoryDataEntriesCtrl($scope, $q, DataEntry, DTOptionsBuilder, DTColumnBuilder) {
+    var vm = this;
+    vm.inventory = $scope.inventory;
+    var options = DTOptionsBuilder.fromFnPromise(function() {
+      var deferred = $q.defer();
+      DataEntry.get({inventory_id: vm.inventory.id}).$promise.then(function(results) {
+        deferred.resolve(results.data_entries);
+      }, deferred.reject);
+      return deferred.promise;
+    });
+
+    //HACK: activate datatables.buttons plugin, we'll be able to get rid of this once we migrate to angular-datatables 0.3.x
+		options.withButtons = function(buttonsOptions) {
+			var options = this;
+			var buttonsPrefix = 'B';
+			options.dom = options.dom ? options.dom : $.fn.dataTable.defaults.sDom;
+			if (options.dom.indexOf(buttonsPrefix) === -1) {
+				options.dom = buttonsPrefix + options.dom;
+			}
+			options.buttons = buttonsOptions;
+			return options;
+		};
+    
+    vm.dtOptions = options.withPaginationType('full_numbers').withButtons([
+      'columnsToggle'
+    ]);
+    vm.dtColumns = [
+      DTColumnBuilder.newColumn('general_data_question.data_type').withTitle('Data Type'),
+      DTColumnBuilder.newColumn('general_data_question.data_capture').withTitle('Data Capture'),
+      DTColumnBuilder.newColumn('data_entry_question.who_enters_data').withTitle('Who Enters Data'),
+      DTColumnBuilder.newColumn('data_access_question.who_access_data').withTitle('Who Access Data'),
+    ];
```

Comment:
We discussed this yesterday, turns out you shouldn't be able to create an entry without product name.